doc/nn_geterror.txt

```diff
@@ -0,0 +1,51 @@
+nn_geterror(3)
+===========
+
+NAME
+----
+nn_geterror - retrieve the current error for an endpoint
+
+
+SYNOPSIS
+--------
+*#include <nanomsg/nn.h>*
+
+*int nn_geterror (int s, int ep);*
```

Comment:
The other examples single quote (with ') the named arguments.  While I'm not a big fan of whatever markup this style is, we should be consistent for now.